File: Iniciante/URI 1963.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
A = tonumber(line[1])
B = tonumber(line[2])

aumento = (B * 100.0 / A) - 100.0
print(string.format('%.2f%%', aumento))

File: Iniciante/URI 1963.py
aumento = None
A = None
B = None
lista = None

def read_line():
  try:
    # read for Python 2.x
    return raw_input()
  except NameError:
    # read for Python 3.x
    return input()

lista = read_line().split(" ")
A = float((lista[0]))
B = float((lista[1]))
aumento = (B * 100) / A - 100
print(str("{:0.2f}".format(aumento)) + str("%"))